analysisservice/src/main/java/nu/yona/server/analysis/service/AnalysisEngineService.java

```diff
@@ -41,13 +40,40 @@
 @Service
 public class AnalysisEngineService
 {
+	private class UserAnonymizedEntityHolder
```

Comment:
I bet you've seen the purpose when you looked further down the code. I've moved the inner class to the end, along with ``ActivityPayload``.